hopsworks-common/src/main/java/io/hops/hopsworks/common/jobs/spark/SparkJobConfiguration.java

```diff
@@ -267,6 +267,18 @@ public JobType getJobType() {
       return JobType.SPARK;
     }
   }
+  
+  @Override
+  @XmlElement(name="jobTypeName")
```

Comment:
What's the difference between JobType and JobTypeName?

Reply:
JobTypeName is the "display name" of the type. Because a type can be `beam_flink_k8s`, but it's ugly for the client. We could argue that the display name is the responsibility of the client, but in this case this field is not persisted and our front-end is way too spaghetti to have another lookup table.